src/app/portlets/content-types/fields/content-type-fields-row/content-type-fields-row.component.ts

```diff
@@ -33,7 +36,7 @@ export class ContentTypeFieldsRowComponent extends BaseComponent {
      * Remove a field
      * @param field field to remove
      */
-    removeField(field: Field): void {
+    removeFieldHandler(field: Field): void {
```

Comment:
In the case of the method to call with the event and according to angular styleguide you should use ```onRemoveField()``` as a method name: https://angular.io/guide/styleguide#dont-prefix-output-properties